web/src/main/angular/src/app/core/components/main-contents/index.ts

```diff
@@ -7,6 +7,7 @@ import { ServerMapModule } from 'app/core/components/server-map';
 import { MainContentsContainerComponent } from './main-contents-container.component';
 import { HelpViewerPopupModule } from 'app/core/components/help-viewer-popup';
 import { RealTimeModule } from 'app/core/components/real-time';
+import { SideBarModule } from '../side-bar';
```

Comment:
Use absolute path instead.
```suggestion
import { SideBarModule } from 'app/core/components/side-bar';
```